Custom Health Recording Page & Fixed Fallback Images

bweam opened the pull request:
**Notes:**

- Added alert messages when creating custom health record is successful/unsuccessful
- Connected to backend
- Image handling
- Fixed fallback images for components
- Moved CustomHealthRecordingScreen to folder

bweam committed:
feat: handle image upload & added toast message

bweam committed:
fix: image handling & added alt to FallbackImage

bweam committed:
wip: backend connection

bweam committed:
feat: added FallbackImageLarge for large images

bweam committed:
fix: changed fallbackimage to correct size

bweam committed:
feat: added alerts

bweam committed:
fix: fixed FallbackImage sizes

bweam committed:
refactor: moved CustomHealthRecordingScreen to folder

bweam committed:
fix: renamed fieldName to name

bweam committed:
refactor: removed unused code

pointmekin commented on src/screens/healthRecord/CustomHealthRecordingScreen.tsx:
```diff
@@ -97,37 +118,98 @@ const CustomHealthRecordingScreen = () => {
     return false;
   };
 
+  const handleSubmit = async () => {
+    const uploadImage = customImage
+      ? customImage.assets
+        ? customImage.assets[0]
+        : undefined
+      : undefined;
```
Suggested shorter version. But no worries tho. Yours is ok as well.

```
const uploadImage = customImage && customImage.assets
        ? customImage.assets[0]
        : undefined
```

bweam committed:
refactor: refactor handleSubmit

bweam committed:
rebase

bweam committed:
fix & refactor: changed get to post and fix styling of components

bweam committed:
fix: merge conflicts

bweam committed:
refactor: removed temporary navigation

bweam commented:
I did some further changes. I fixed Pae's endpoint from get to post, refactored some styling, removed temporary navigation & made successful creation of health recording navigate correctly back to Health Recordings page

pointmekin commented:
> I did some further changes. I fixed Pae's endpoint from get to post, refactored some styling, removed temporary navigation & made successful creation of health recording navigate correctly back to Health Recordings page

Lgtm. Merge loey kubb
